style.css

```diff
@@ -591,10 +630,9 @@ filter: invert(20%);
 .what_to_bring-container{

```

Comment:
Instead of having two colours (orange and green gradients) touching each other, a white background was used to make the design less colourful and more refined and elegant looking